scripts/stablex/claim_withdraw.js

```diff
@@ -26,7 +26,7 @@ module.exports = async (callback) => {
     const token = await ERC20.at(token_address)
 
     const balance_before = await token.balanceOf(withdrawer)
-    await instance.withdraw(token_address, { from: withdrawer })
+    await instance.withdraw(token_address, withdrawer, { from: withdrawer })
```

Comment:
the `from:` no longer needed

Reply:
I think the most common use case is that the withdrawer wants to withdraw for himself and not the accounts[0]

But I could make the transaction sender configurable so that it is an input of the script and that the withdrawer is the default sender, if no parameter is specifying the provider.

Wdyt?